src/components/room-metadata-form.yml

```diff
@@ -1,3 +1,9 @@
 roomNameRequired:
   en: Please specify a room name
   de: Bitte geben Sie einen Raumnamen an
+accessLevelInfo:
+  en: Public rooms can be viewed by all users, while private rooms can only be viewed by invited members.
+  de: Öffentliche Räume sind für alle Benutzer sichtbar, während private Räume sind nur für Mitglieder sichtbar.
```

Comment:
If you want to have a verb in the "während" sentence, you'd have to put it in the end. But a shorter version could also be: "Öffentliche Räume sind für alle Benutzer sichtbar, private Räume nur für Mitglieder."

Reply:
Perfect, updated! Used "Raummitglieder" here too for clarity.

Reply:
:+1: